listenbrainz/spark/handlers.py

```diff
@@ -48,9 +49,12 @@ def handle_user_artist(data):
     if is_new_user_stats_batch():
         notify_user_stats_update()
     current_app.logger.debug("inserting stats for user %s", musicbrainz_id)
-    artists = data['artist_stats']
-    artist_count = data['artist_count']
-    db_stats.insert_user_stats(user['id'], artists, {}, {}, artist_count)
+
+    # delete extra data
+    del data['musicbrainz_id']
```

Comment:
again, let's not modify data passed into the function, instead copy the data as you need.